src/Microsoft.ML.Core/Utilities/ResourceManagerUtils.cs

```diff
@@ -113,8 +113,7 @@ public async Task<ResourceDownloadResults> EnsureResourceAsync(IHostEnvironment
                 await DownloadFromUrlWithRetryAsync(env, ch, absoluteUrl.AbsoluteUri, fileName, timeout, filePath), absoluteUrl.AbsoluteUri);
         }
 
-        private async Task<string> DownloadFromUrlWithRetryAsync(IHostEnvironment env, IChannel ch, string url, string fileName,
-            int timeout, string filePath, int retryTimes = 5)
+        private async Task<string> DownloadFromUrlWithRetryAsync(IHostEnvironment env, IChannel ch, string url, string fileName, int timeout, string filePath, int retryTimes = 5)
```

Comment:
Can you please revert this? This is not good for a few reasons:
1. Your change does not change behavior in any way. It just puts the whole function on a different line.
2. It does not improve formatting to make it more readable either because the previous formatting is more readable. The deliberate word wrap exists to prevent horizontal scrolling while reading code.
3. It makes reviewers pause and read more carefully as to whether you changed something significant.